TestFoundation/TestNSOrderedSet.swift

```diff
@@ -135,4 +149,185 @@ class TestNSOrderedSet : XCTestCase {
         XCTAssertEqual(set.count, 2)
         XCTAssertEqual(set.indexOfObject("baz" as NSString), 1)
     }
-}
\ No newline at end of file
+
+    func test_IsEqualToOrderedSet() {
+        let set = NSOrderedSet(array: ["foo", "bar", "baz"].bridge().bridge())
+        let otherSet = NSOrderedSet(array: ["foo", "bar", "baz"].bridge().bridge())
+        let otherOtherSet = NSOrderedSet(array: ["foo", "bar", "123"].bridge().bridge())
+        XCTAssert(set.isEqualToOrderedSet(otherSet))
+        XCTAssertFalse(set.isEqualToOrderedSet(otherOtherSet))
+    }
+
+    func test_Subsets() {
+        let set = NSOrderedSet(array: ["foo", "bar", "baz"].bridge().bridge())
+        let otherOrderedSet = NSOrderedSet(array: ["foo", "bar"].bridge().bridge())
+        let otherSet = Set(["foo".bridge(), "baz".bridge()])
+        let otherOtherSet = Set(["foo".bridge(), "bar".bridge(), "baz".bridge(), "123".bridge()])
+        XCTAssert(otherOrderedSet.isSubsetOfOrderedSet(set))
+        XCTAssertFalse(set.isSubsetOfOrderedSet(otherOrderedSet))
+        XCTAssertFalse(set.isSubsetOfSet(otherSet))
```

Comment:
Casting Set to Set<NSObject> did the trick. Something must be up with type inference here?